topics/statistics/tutorials/classification-machinelearning/tutorial.md

```diff
@@ -0,0 +1,548 @@
+---
+layout: tutorial_hands_on
+
+title: 'Classification in Machine Learning'
+zenodo_link: https://zenodo.org/record/...
+questions:
+- What is classification and how we can use classification techniques?
+objectives:
+- Learn classification background
+- Apply classification based machine learning algorithms
+- Learn ... (Simon)
```

Comment:
Why is Simon's name present?

Reply:
I am not sure why classification should be explained again while doing deep learning? They should be separate